pgml-sdks/pgml/src/queries.rs

```diff
@@ -134,8 +134,10 @@ WHERE
   AND id NOT IN (
```

Comment:
Exists is more idiomatic
```suggestion
  AND NOT EXISTS (
```

Reply:
The query planner should optimize this as equivalent, but you could also do a `LEFT OUTER JOIN "table" ON chunk_id = ... WHERE "table".id IS NULL` which will select only the missing rows from "table", if the subquery shows up as an expensive component.